patches/chrome-browser-resources-settings-BUILD.gn.patch

```diff
@@ -1,19 +1,12 @@
 diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
-index fa23f4af6c3904ffe44430d870fedb6e057ab94c..d0dc53be4f529e1da3d18358c4b8c6bf9a41d4ac 100644
+index 2e218dc46b90016e180da9b36232f47a1779c58e..750a87d8deefa9d1da807ca1442177750a1794d0 100644
 --- a/chrome/browser/resources/settings/BUILD.gn
 +++ b/chrome/browser/resources/settings/BUILD.gn
-@@ -168,6 +168,8 @@ ts_library("build_ts") {
-     # writing, unfortunately).
-     extra_deps += [ "//ash/webui/common/resources:generate_definitions" ]
+@@ -477,4 +477,7 @@ build_webui("build") {
+       "lazy_load.rollup.js|lazy_load.js",
+     ]
    }
-+
-+  in_files += brave_deps_chrome_browser_resources_settings_in_files extra_deps += brave_deps_chrome_browser_resources_settings_extra_deps
- }
- 
- generate_grd("build_grd") {
-@@ -256,4 +258,5 @@ generate_grd("build_grd") {
-     manifest_files =
-         filter_include(get_target_outputs(":build_ts"), [ "*.manifest" ])
-   }
-+  deps += [ "//brave/browser/resources/settings:resources" ]
++  extra_build_ts_in_files = brave_deps_chrome_browser_resources_settings_in_files
```

Comment:
these can all go on one line

Reply:
Fixed in 7c6fd080ddde271c01d91241123c2faf147357ad.